docs/input/docs/reference/build-servers/gitlab.md

```diff
@@ -7,3 +7,27 @@ Description: Details on the GitLab CI support in GitVersion
 To use GitVersion with GitLab CI, either use the [MSBuild
 Task](/docs/usage/msbuild) or put the GitVersion executable in your
 runner's `PATH`.
+
+A working example of integrating GitVersion with GitLab is maintained in the project [Utterly Automated Software and Artifact Versioning with GitVersion][1]
+
+Here is a summary of what it demonstrated (many more details in the [README.md][2])
+
+- Is a working example know as a [Guided Explorations (GE Manifesto)][3] - so job logs and package artifacts can be reviewed. The project can also be imported to your own GitLab group or instance.
+- IMPORTANT: It demonstrates how to override GitLab CI's default cloning behavior so that GitVersion can do a dynamic copy. This best practice demonstrates the best way to do this while avoiding a double-cloning of the project (once by GitLab Runner and once by GitVersion)
+- Implements GitVersion as a CI CD Extension that can be reused across many projects on the same GitLab instance using includes.
+- Implements GitVersion as a single job that runs the GitVersion container and passes the version number downstream into both PIPLINE and JOB level variables, which means...
+- It can be used with ANY coding language, framework or packaging engine.
+- Generates example packaged artifacts:
+  - Two ways of building Sem Versioned [NuGet packages][4] (msbuild-ish and nuget.exe-ish) and uploads them and tests them from a [GitLab NuGet repository][4].
+  - A Sem Versioned [GitLab "Generic Package"][5]
+  - A Sem Versioned docker container and uploads to [GitLabs Container Registry][6].
+- It creates a Sem Versioned [GitLab Release](GitLab Releases Feature Help) and Git tag using the [GitLab release-cli][7] and links the generic package as evidence.
+
+### References:
+[1]: https://gitlab.com/guided-explorations/devops-patterns/utterly-automated-versioning/
```

Comment:
When all references are named instead of numbered, please sort them alphabetically.

```suggestion
[utterly-automated]: https://gitlab.com/guided-explorations/devops-patterns/utterly-automated-versioning/
```